dev-haskell/lifted-async/lifted-async-0.10.2.3.ebuild

```diff
@@ -1,16 +1,15 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
-# ebuild generated by hackport 0.6.7.9999
+# ebuild generated by hackport 0.8.0.0
```

Comment:
Did you get hackport working with ghc-9.2?

Reply:
@hololeap yes... and working version is in pull request, you've seen it, I've made ebuild for myself for now.

but regarding hackport changes - not sure what to do, I can keep working on things to get better and merge-able, like drop cabal submodules etc if nobody else is on this task but I can't promise fast results, especially in this month

Reply:
Ok, I wasn't sure if you had made any other changes on your end, since that PR is still in draft form. Obviously there aren't a lot of people working on it right now, and we're all strapped for time, so _any_ progress you make on it is appreciated. I have a half-decent understanding of hackport's code, so I should be able to integrate any changes you make. Thanks for working on it.